docs/src/pages/ui/primitives/tabs/demo.tsx

```diff
@@ -48,7 +48,7 @@ export const TabsDemo = () => {
 export const ControlledTabExample = () => {
   const [index, setIndex] = React.useState(0);
   return (
-    <Tabs currentIndex={index} onChange={(i) => setIndex(i)}>
+    <Tabs currentIndex={index} onChange={(i) => setIndex(i as number)}>
```

Comment:
I believe `React.useState<number>(0)` will help the inference so that `setIndex` doesn't require casting 🤔

Reply:
I thought so too but TS disagreed. 🤷‍♂️

Reply:
Typing the parameter may be better than casting?

`<Tabs currentIndex={index} onChange={(i: number) => setIndex(i)}>`

Reply:
@zchenwei Good call, updated.